index.html

```diff
@@ -5,40 +5,265 @@
+          <img class="portrait1" src="images/Snapshoot Portfolio (1).png"/>
+     <div class="description1">
+       <h2>
+        <strong class="stronger"> Multi-Post
+        <br>
+        Stories
+    </strong>
+      </h2>
+    
+        <div class="canopy">
+            <ul>
+                <li><div class="canopy1">CANOPY</div></li>
+             
+                <li> Full Stack Dev</li>
+                
+                <li>2015</li>
+            </ul>
+          
+        </div>                
+        <p>
+            A daily selection of privately <br>
+            personalized reads; no accounts or <br>sign-ups required.
+        </p>
+        <div class="language">
+          <ul>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+        </div>
+
+        
+        <div class="project_tag"><button>See project</button></div>
+       
+    </div>
+</div>
+
+</div>
+<div class="work_item1">
+    <div class="information1">
+          <img class="portrait1" src="images/Snapshoot Portfolio (2).png"/>
+     <div class="description1">
+       <h2>
+        <strong class="stronger">Tonic</strong>
+      </h2>
+    
+        <div class="canopy">
+            <ul>
+                <li><div class="canopy1">CANOPY</div></li>
+                 
+                <li> Full Stack Dev</li>
+                
+                <li>2015</li>
+            </ul>
+          
+        </div>                
+        <p>
+            A daily selection of privately <br>
+            personalized reads; no accounts or <br>sign-ups required.
+        </p>
+        <div class="language">
+          <ul>
+             <li>HTML</li>
+             <li>CSS</li>
+             <li>JavaScript</li>
+          </ul>
+        </div>
+
+        
+        <div class="project_tag"><button>See project</button></div>
+       
+    </div>
+</div>
+
+</div>
+<div class="work_item1">
+    <div class="information1">
+          <img class="portrait1" src="images/Snapshoot Portfolio (3).png"/>
+     <div class="description1">
+       <h2>
+            <strong class="stronger">Multi-Post
+                <br>
+                Stories
+            </strong>
+      </h2>
+    
+        <div class="canopy">
+            <ul>
+                <li><div class="canopy1">CANOPY</div></li>
+             
+                <li> Full Stack Dev</li>
+                
+                <li>2015</li>
+            </ul>
+          
+        </div>                
+        <p>
+            A daily selection of privately <br>
+            personalized reads; no accounts or <br>sign-ups required.
+        </p>
+        <div class="language">
+            <ul>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+          </div>
+  
+
+        
+        <div class="project_tag"><button>See project</button></div>
+       
+    </div>
+</div>
+
+</div>
+    </div>
+    <div class="aboutme">
```

Comment:
- [ ] please like the previous reviewer noted, The container of the section is a section tag. a section is a semantic element for creating standalone sections in a web page. this will be of great use. kindly remove the header tag from the about section as it represents introductory content, typically a group of introductory or navigational aids.
eg 
```
<section>
      .. about section
</section>
```

Reply:
Dear sir, thanks for your fruitful comments. So should I use a section tag instead of a div name called 'aboutme'?